cmd/internal/shared/gatewayserver/config.go

```diff
@@ -66,4 +66,6 @@ var DefaultGatewayServerConfig = gatewayserver.Config{
 		Listen:                 ":1887",
 		ListenTLS:              ":8887",
 	},
+
+	DisconnectTTL: 48 * time.Hour,
```

Comment:
No new line, and also probably sits better under `UpdateConnectionStatsDebounceTime`.